TEMPLATE

```diff
@@ -0,0 +1,36 @@
+EXAMPLE:
```

Comment:
This would mean people are still submitting individual notes and not editing a combined document correct?

Reply:
No, That would just be the format for each persons contributions to the Notes file. A begin to end block with each person appending their notes to the end of the file